modules/net-lb-ext/main.tf

```diff
@@ -34,7 +34,7 @@ resource "google_compute_forwarding_rule" "default" {
   provider = google-beta
   project  = var.project_id
   region   = var.region
-  name = (
+  name = var.override_name != null ? var.override_name : (
```

Comment:
I'm not following the logic:

- if you have multiple forwarding rules, the override will work for one and fail creation of the others?
- if you have a single one with an empty key, the plain name will be used which is probably what you want?

I think you need to do the same for forwarding rules as you did for health checks, allowing the override for each rule.

Reply:
See changes. Allowed the override to be used for each rule. Override would now not fail where there are multiple forwarding rules in the list